examples/mikrobus/eth.mnfs

```diff
@@ -0,0 +1,51 @@
+;
+; ETH Click
+; https://www.mikroe.com/eth-click
+;
+; Copyright 2020 BeagleBoard.org Foundation
+; Copyright 2020 Texas Instruments
+;
+
+[manifest-header]
+version-major = 0
+version-minor = 2
+
+[interface-descriptor]
+vendor-string-id = 1
+product-string-id = 2
+
+[string-descriptor 1]
+string = Mikroelectronika
+
+[string-descriptor 2]
+string = Eth Click
+
+[bundle-descriptor 1]
+class = 10
+
+[cport-descriptor 1]
+bundle = 1
+protocol = 2
+
+[cport-descriptor 2]
+bundle = 1
+protocol = 2
+
+[cport-descriptor 3]
+bundle = 1
+protocol = 11
+
+[mikrobus-descriptor]
```

Comment:
This is a nice approach, moving this to a new descriptor from manifest header will make the manifest source look better.